pyjobshop/ortools/VariablesManager.py

```diff
@@ -246,10 +249,18 @@ def _make_task_variables(self) -> list[TaskVar]:
                 ub=min(task.latest_end, data.horizon),
                 name=f"{name}_end",
             )
-            interval = model.NewIntervalVar(
-                start, duration, end, f"interval_{task}"
+            is_present = (
+                model.new_constant(1)
+                if task.required
+                else model.new_bool_var(f"{name}_is_present")
+            )
+            # TODO Do we need to use new_interval_var if the task is required?
+            interval = model.new_optional_interval_var(
```

Comment:
check this implementation

Reply:
I don't think it matters since they all construct an IntervalVar.

Reply:
I hope presolve solves this by identifying that the enforcement literal is always true.